node/clients/GoogleGeolocation.ts

```diff
@@ -6,10 +6,18 @@ export class GoogleGeolocation extends ExternalClient {
   }
 
   public reverseGeocode = (lat: string, lng: string, apiKey: string) => {
+    return this.http.get(`/geocode/json?latlng=${lat},${lng}&key=${apiKey}`, {
+      metric: 'google-reverse-geocode-country',
+      timeout: 3000,
+    })
+  }
+
+  public suggestAddresses = (searchTerm: string, apiKey: string) => {
+    const input = encodeURI(searchTerm)
     return this.http.get(
-      `/geocode/json?latlng=${lat},${lng}&key=${apiKey}`,
+      `/place/autocomplete/json?input=${input}&key=${apiKey}`,
       {
-        metric: 'google-reverse-geocode-country',
+        metric: 'google-suggest-addresses',
         timeout: 3000,
       }
     )
```

Comment:
This is not being used and should be removed (unless the `@googlemaps/google-maps-services-js` client get discarded)